farmdata2_modules/fd2_tabs/fd2_barn_kit/seedingReport.html

```diff
@@ -733,6 +733,7 @@ <h1 class="text-center">Seeding Report</h1>
                     for(i=0; i < this.userToIDMap.size; i++){
                         userNameArray.push(userKeyIterator.next().value)
                     }
+                    userNameArray.splice(0, 1)
```

Comment:
Should this be here? Looks like it will be cutting out a name?

Reply:
This has now been resolved, sorry about that.